views/pages/customError.ejs

```diff
@@ -0,0 +1,15 @@
+<!-- views/pages/error.ejs -->
```

Comment:
Let's add this functionality into the main error page

Reply:
done.  Redid your token error handling to use it, and remove the route for /error